<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta name="zincbinder" content="" />
<title>ZincBinder</title>
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="style.css" />
</script>
</head>
<body>
<div id="wrapper">
<div id="header">
<div id="logo"><img src="images/zinclogo.png" width="406" height="130" /></div>
<div id="slogan"><img src="images/logo.png" width="119" height="128"></div></div>
<div id="menu">
  <p><a href="index.html">Home</a><a href="prediction.html">Prediction</a><a href="algorithm.html">Algorithm</a><a href="team.html">Developers</a><a href="help.html">Help</a><a href="contact.html">Contact</a></p>
  </li></ul></div>
<div id="slider"><table>
  <tr><td height="28" align="center"><h4><font color="#EE0000">General information:</font></h4></td></tr>
  
  <tr>
    <td height="28" align="justify">Zincbinder   is a simple and accurate support vector machine based prediction system, which takes a protein sequence as an input and predicts zinc metal ion binding sites. User can perform the predictions of their choice and receive the results of the prediction via e-mail or the web.</td></tr>
 
  <tr><td height="28"><h5><font color="#EE0000">Users can follow the following steps while using the ZincBinder:-</font></h5></td></tr> 
  
  <tr><td height="28"><h5>Sequence Name:</h5></td></tr>
  <tr><td height="28" align="justify">In this area,the user can submit the name of the query sequence. This is optional and will not affect the prediction</td></tr>
<tr><td height="28"><h5>E-mail Address:</h5></td></tr>
<tr><td height="28" align="justify">Depending upon the option which will be opted for prediction or number of queries in que, time required to serve the query will vary from 10 seconds to more than 5 minutes. Instead of waiting till the prediction is complete, this option provide oppurtunity to put the email-id of user to the server. As the prediction is complete, user will receive an email to intimate the same. Only alphabets or numerics or '_' should be there in email-id.</td></tr>
   
  <tr><td height="28"><h5>Input Sequence :</h5></td></tr>
    
<tr>
  <td align="justify"><p>The user can paste query sequence in fasta format in the provided inbox.The server also has the facility for uploading the local sequence files or by browsing the disk . Amino acid sequences must be entered in the one-letter code.All the non standard characters will be ignored from the sequence.A sample of submission form with lebeled fields is shown below.</p></td></tr>
  <tr><td align="center"><img src="images/prediction.jpg" width="700" height="500" /></td></tr>
    <tr><td height="28"><h5>SVM Threshold :</h5></td></tr>
  <tr>
    <td 
  align="justify">The method is based on Support Vector Machine (SVM). Depending upon the threshold value which user choses, SVM will classify the ziinc binding sites and non zinc binding sites in protein sequences. The default threshold is 0. If user want less sensitivity but more specificity, then higher threshold value should be specified, but if opposite is anticipated then lower threshold value should be choosen. So, the expected outcome will depends on the trade-off between sensitivity and specificity.</td></tr>

<tr><td height="28"><h5>Prediction Result:</h5></td></tr>
<tr>
  <td height="28" align="justify">After analysis the prediction results are shown in user-friendly format.The results provides information about summary of submitted sequence and final prediction results.Prediction results provides information in four tabulated column showing query sequence, zinc binding residue and their sequence position  and corresponding zinc binding score.A sample of prediction output is shown below.</td></tr>
  <tr><td align="center"><img src="images/prediction_result.jpg" width="700" height="500" /></td></tr>
<tr><td height="28"><h5>Standalone:</h5></td></tr>

<tr>
  <td align="justify">A standalone version of ZincBinder prediction software be downloaded <font color="#EE0000"><a href="znbinder_standalone.tar.gz">here</a></font> . The software has been only tested on Linux. Please note that you should read the license terms given in the <font color="#EE0000"><a href="standalone.html">README</a></font> file if you wish to incorporate ZincBinder in another program .</td></tr>
<tr><td><h5>Dataset</h5></td></tr>
<tr>
  <td>The complete list of the data sets used for ZincBinder evaluation can be downloaded <font color="#EE0000"><a href="supplementry.html">here</a></font> .</td></tr>
</table>

</div>
<div id="page-bottom"><div id="page-bottom-content"><p>ZincBinder is maintained by Department of Biophysics, University of Delhi South Campus. We acknowledge your comments or contribution to this resources. Please send your sugestions to Dr. Manish Kumar (Email:manish at south dot du dot ac dot in)</p></div>
<br class="clearfix"/>
</div>
</div>
</body>
</html>
